core-bundle/src/Resources/public/core.js

```diff
@@ -1329,6 +1329,30 @@ var Backend =
 		});
 	},
 
+	/**
+	 * Copy the interactive help from the button image
+	 */
+	copyInteractiveHelpFromButtonsImage: function (buttons) {
```

Comment:
I would prefer `copyInteractiveHelpFromButtonImage` like you wrote in the doc comment.

Reply:
Well, now we can discuss if it's wrong in the docs or if we should rename the method. It really copies the alt text from multiple buttons - what's convention?

Reply:
I guess both "the button image" and "the button's image" are valid, however, with 250.000 Google results to 10.000 Google results, "the button image" seems to be more common. Also, it is an elliptical sentence and "copy interactive help from button's image" does not sound correct to me. But maybe @blairwinans can help us out here?

Reply:
Oh wow, this is a tough one. My preference is `copyInteractiveHelpFromButtonImage` because there is no way to add an apostrophe to show possession. Since it takes an array you could argue that it should just be `copyInteractiveHelpFromButtons` since that is what is being passed, but if you feel the need to include the Image, I would make it singular with `copyInteractiveHelpFromButtonImage`. That my two cents!

Reply:
Okay, I'm fine with `copyInteractiveHelpFromButtonImage` - it was even my first idea before the previous committed one 🙃